x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -193,10 +190,9 @@ export class TOCEntryActionsPopover extends Component<Props, State> {
           toolTipContent: this.state.isFeatureEditingEnabled
             ? null
             : i18n.translate('xpack.maps.layerTocActions.editFeaturesTooltip.disabledMessage', {
-                defaultMessage:
-                  'Edit features only supported for document layers without clustering, term joins, time filtering, or global search.',
+                defaultMessage: `To enable edit features, select 'Use vector tiles' in 'Scaling'.`,
```

Comment:
Lets change to 'Edit features is only supported for layers without clustering and term joins'